debian/control

```diff
@@ -10,7 +10,7 @@ Package: labgrid
 Architecture: any
 Pre-Depends: dpkg (>= 1.16.1), python3, ${misc:Pre-Depends}
 Depends: ${python3:Depends}, ${misc:Depends}, ${shlibs:Depends}
-Recommends: openssh-client, microcom, socat, sshfs, rsync
+Recommends: openssh-client, microcom, socat, sshfs, rsync, pmount
```

Comment:
Why do you prefer `pmount` over `udiskctl`?

Reply:
Will resubmit with udisksctl